muze/src/db/musicBattlesLikes.ts

```diff
@@ -0,0 +1,64 @@
+import SupabaseClient from './SupabaseClient';
```

Comment:
replace this with import from `muze/src/lib/supabase/supabase.ts`

SupabaseClient has been removed.